src/app/pubspec.lock

```diff
@@ -205,6 +212,20 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "9.2.0"
+  geocoding:
+    dependency: "direct main"
+    description:
+      name: geocoding
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "2.0.1"
+  geocoding_platform_interface:
+    dependency: transitive
+    description:
+      name: geocoding_platform_interface
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "2.0.1"
   google_geocoding:
     dependency: "direct main"
     description:
```

Comment:
is google geocoding still used in other parts of the application? If not we can remove it

Reply:
Yea, its still used in other parts, I can remove it completely if you want,

Reply:
we can keep it as it is for now